apis/r/configure

```diff
@@ -45,10 +51,10 @@ have_tiledb="false"
 ## check for pkg-config and use it to inquire about tiledb build options
 pkg-config --version >/dev/null 2>&1
 if [ $? -eq 0 ]; then
-    pkg-config --exists tiledb
+    pkg-config --exists tiledb tiledbsoma
     if [ $? -eq 0 ]; then
-        pkgcflags=`pkg-config --cflags tiledb`
-        pkglibs=`pkg-config --libs tiledb`
+        pkgcflags=`pkg-config --cflags tiledb tiledbsoma`
+        pkglibs=`pkg-config --libs tiledbsoma tiledb`
```

Comment:
All this is in the configure script for R when we decided overall to not use a shared library with R.  So I think we should also not ask `pkg-config` about the (shared, system) library for tiledbsoma.